.github/workflows/build-test.yml

```diff
@@ -210,3 +214,33 @@ jobs:
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
                 TARGET_DOCKER_TEST_IMAGE: ${{ matrix.image }}
               run: ./ci/test.sh
+
+    display_code_coverage:
+        needs: test-mac-required
+        name: Display code coverage
+        runs-on: macos-latest
+        strategy:
+          matrix:
+            cloud: [ 'AWS' ]
+        steps:
+          - uses: actions/checkout@v1
+          - uses: actions/setup-node@v1
+            with:
+              node-version: '12.x'
+          - uses: actions/setup-python@v1
+            with:
+              python-version: '3.7'
+          - name: Download Build Artifacts
+            uses: actions/download-artifact@v1
+            with:
+              name: artifacts
+          - name: Install Homebrew Bash
+            shell: bash
+            run: brew install bash
+          - uses: actions/download-artifact@master
+            with:
+              name: codeCoverage
+              path: .nyc_output
+          - name: Display code coverage
+            shell: bash
+            run: ./ci/displayCodeCoverage.sh
```

Comment:
can we display the result without script?

Reply:
yes